src/vsr/replica.zig

```diff
@@ -76,7 +76,7 @@ pub fn ReplicaType(
         const Self = @This();
 
         const Journal = vsr.JournalType(Self, Storage);
-        const Clock = vsr.Clock(Time);
```

Comment:
These two lines are the whole motivation for the PR, it always seemed odd that they are inconsistent.